File: ployglots/mta-ployglot-pkg-vault/packages/ts/examples/sdk/04-describe-config.mjs
import { VaultFileSDK } from '@polyglot/vault-file';

const sdk = VaultFileSDK.create().withEnvPath(process.argv[2] ?? '.env').build();
sdk.loadConfig();
const r = sdk.describeConfig();
console.log(JSON.stringify(r.data, null, 2));

File: ployglots/mta-ployglot-pkg-vault/packages/py/examples/sdk/04_describe_config.py
import sys

from polyglot_vault_file import VaultFileSDK

env_path = sys.argv[1] if len(sys.argv) > 1 else ".env"
sdk = VaultFileSDK.create().with_env_path(env_path).build()
sdk.load_config()
r = sdk.describe_config()
print(r.data.model_dump_json(indent=2, by_alias=True))
